src/Valet/Commands/Common.cs

```diff
@@ -5,16 +5,9 @@ namespace Valet.Commands;
 
 public static class Common
 {
-    public static Command AppendCommonOptions(Command command)
-    {
-        command.AddGlobalOption(
-            new Option<DirectoryInfo>(new[] { "--output-dir", "-o" })
-            {
-                IsRequired = true,
-                Description = "The location for any output files."
-            }
-        );
 
+    public static Command AppendTransformerOptions(Command command)
```

Comment:
```suggestion
    public static void AppendTransformerOptions(this Command command)
```

What do you think about making these extension methods? It might be nice to be able to use a more fluent syntax when invoking these?

If we did this consistently then we could write stuff like this:

```csharp
    public static void AppendCommonOptions(this Command command)
    {
        command.AppendGeneralRequiredOptions()
               .AppendTransformerOptions()
               .AppendGeneralOptions();
    }
```

Reply:
Love it, although I had to make the individual methods return a Command so that we could stack them like you suggested.